packages/google_maps_flutter/example/integration_test/src/tiles_inspector.dart

```diff
@@ -204,6 +207,271 @@ void runTests() {
+      expect(color1.color, color2.color);
+      expect(
+        color1.startPoint,
+        moreOrLessEquals(color2.startPoint, epsilon: floatTolerance),
+      );
+    }
+
+    expect(gradient1.colorMapSize, gradient2.colorMapSize);
+  }
+
+  void expectHeatmapEquals(Heatmap heatmap1, Heatmap heatmap2) {
+    expectHeatmapDataMoreOrLessEquals(heatmap1.data, heatmap2.data);
+    expectHeatmapGradientMoreOrLessEquals(heatmap1.gradient, heatmap2.gradient);
+
+    // Only Android supports `maxIntensity`
+    // so the platform value is undefined on others.
+    bool canHandleMaxIntensity() {
+      return Platform.isAndroid;
+    }
+
+    // Only iOS supports `minimumZoomIntensity` and `maximumZoomIntensity`
+    // so the platform value is undefined on others.
+    bool canHandleZoomIntensity() {
+      return Platform.isIOS;
+    }
+
+    if (canHandleMaxIntensity()) {
+      expect(heatmap1.maxIntensity, heatmap2.maxIntensity);
+    }
+    expect(
+      heatmap1.opacity,
+      moreOrLessEquals(heatmap2.opacity, epsilon: floatTolerance),
+    );
+    expect(heatmap1.radius, heatmap2.radius);
+    if (canHandleZoomIntensity()) {
+      expect(heatmap1.minimumZoomIntensity, heatmap2.minimumZoomIntensity);
+      expect(heatmap1.maximumZoomIntensity, heatmap2.maximumZoomIntensity);
+    }
+  }
+
+  const Heatmap heatmap1 = Heatmap(
+    heatmapId: HeatmapId('heatmap_1'),
+    data: <WeightedLatLng>[
+      WeightedLatLng(LatLng(37.782, -122.447)),
+      WeightedLatLng(LatLng(37.782, -122.445)),
+      WeightedLatLng(LatLng(37.782, -122.443)),
+      WeightedLatLng(LatLng(37.782, -122.441)),
+      WeightedLatLng(LatLng(37.782, -122.439)),
+      WeightedLatLng(LatLng(37.782, -122.437)),
+      WeightedLatLng(LatLng(37.782, -122.435)),
+      WeightedLatLng(LatLng(37.785, -122.447)),
+      WeightedLatLng(LatLng(37.785, -122.445)),
+      WeightedLatLng(LatLng(37.785, -122.443)),
+      WeightedLatLng(LatLng(37.785, -122.441)),
+      WeightedLatLng(LatLng(37.785, -122.439)),
+      WeightedLatLng(LatLng(37.785, -122.437)),
+      WeightedLatLng(LatLng(37.785, -122.435), weight: 2)
+    ],
+    dissipating: false,
+    gradient: HeatmapGradient(
+      <HeatmapGradientColor>[
+        HeatmapGradientColor(
+          Color.fromARGB(255, 0, 255, 255),
+          0.2,
+        ),
+        HeatmapGradientColor(
+          Color.fromARGB(255, 0, 63, 255),
+          0.4,
+        ),
+        HeatmapGradientColor(
+          Color.fromARGB(255, 0, 0, 191),
+          0.6,
+        ),
+        HeatmapGradientColor(
+          Color.fromARGB(255, 63, 0, 91),
+          0.8,
+        ),
+        HeatmapGradientColor(
+          Color.fromARGB(255, 255, 0, 0),
+          1,
+        ),
+      ],
+    ),
+    maxIntensity: 1,
+    opacity: 0.5,
+    radius: HeatmapRadius.fromPixels(40),
+    minimumZoomIntensity: 1,
+    maximumZoomIntensity: 20,
+  );
+
+  testWidgets('set heatmap correctly', (WidgetTester tester) async {
```

Comment:
What do you think about that the patch which you implement and the test cases together are added?

Reply:
https://github.com/flutter-tizen/plugins/pull/816#issuecomment-2649857786

heatmap does not work in the lwe engine and its implementation is not included in this PR.

Reply:
Yes. So I was curious about your thoughts on adding test cases along with a later patch since we don't know how long it will take to fix LWE.